### PR TITLE
docs: add mentions component filterOption default

### LINKED ISSUE #19235: mentions异步加载输入中文问题
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/mentions-cn/#components-mentions-demo-async](https://ant.design/components/mentions-cn/#components-mentions-demo-async)

### Steps to reproduce
1.中文状态下进入文档异步加载demo进行输入

2.首先输入@

3.输入查询字符串，点击enter

### What is expected?
和正常结果一样，进行异步加载数据

### What is actually happening?
在进入loading状态后返回暂无数据

| Environment | Info |
|---|---|
| antd | 3.23.6 |
| React | x |
| System | win10 |
| Browser | chrome 77 |

### PR DESCRIPTION
**Pull request by shaodahong**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

Fixed #19235 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Doc add mentions component filterOption default |
| 🇨🇳 Chinese | 文档添加  mentions 组件 filterOption 属性的默认值|

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
默认需要触发 filter
ref https://github.com/ant-design/ant-design/issues/19235#issuecomment-542591876

**Comment by shaodahong** [comment]
加个默认值吧

**Comment by zombieJ** [comment]
👌，代码那边是没动只是 prettier 对吧？

File: components/mentions/index.en-US.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ When need to mention someone or something.
 | --- | --- | --- | --- | --- |
 | autoFocus | Auto get focus when component mounted | boolean | `false` | 3.19.0 |
 | defaultValue | Default value | string | - | 3.19.0 |
-| filterOption | Customize filter option logic | false \| (input: string, option: OptionProps) => boolean | - | 3.19.0 |
+| filterOption | Customize filter option logic | false \| (input: string, option: OptionProps) => boolean | `(input, { value }) => value.toLowerCase().indexOf(input.toLowerCase()) !== -1` | 3.19.0 |
```
Comment (zombieJ) [inline components/mentions/index.en-US.md]: 这个太长了，也不是用户关心的。改成 Case Insensitive 吧

Comment (shaodahong) [reply components/mentions/index.en-US.md]: 不区分大小写感觉令人困惑，解释不了中文和拼音之间的联动性，至少实际项目中异步返回数据来说并不是字符的 `indexOf`，就和 `GitHub` 的 `#` 一样

**Pull request closed without merging**